package.json

```diff
@@ -42,10 +42,11 @@
   },
   "homepage": "https://github.com/hmrc/hmrc-frontend#readme",
   "dependencies": {
-    "govuk-frontend": "^5.4.1",
-    "accessible-autocomplete": "^2.0.4"
+    "accessible-autocomplete": "^2.0.4",
+    "govuk-frontend": "^5.4.1"
```

Comment:
I don't know why npm always likes to swap these around in order when I install new stuff